api/envoy/config/route/v3/route_components.proto

```diff
@@ -779,6 +818,11 @@ message RouteAction {
     // :ref:`traffic splitting <config_http_conn_man_route_table_traffic_splitting_split>`
     // for additional documentation.
     WeightedCluster weighted_clusters = 3;
+
+    // A list of selectors that will be iterated in order. If a selector is unable to return an
+    // active cluster, the next selector in the list will be attempted. If an active cluster can't
+    // be found after iterating this list, an error response will be returned to the client.
+    ClusterSelectorRepeatedWrapper cluster_selectors = 35;
```

Comment:
What's the use case @AndresGuedez? Happy to discuss offline on IM as well.

Reply:
I reached out offline. Happy to discuss on Monday.

Reply:
I think this would essentially allow "fallback routing", right? I know we have had a lot of requests for this, but I can't find an issue currently.

Reply:
@mattklein123 right, that's the goal for this change.

Reply:
Agreed that this use case makes sense.